api/helpers/gamestate/moves/points/validate.js

```diff
@@ -0,0 +1,62 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Validate request to play points',
+
+  description: 'Verifies whether a request to make points is legal, throwing explanatory error if not.',
+
+  inputs: {
+    requestedMove: {
+      type: 'ref',
+      description: 'Object containing data needed for current move',
+      required: true,
+    },
+    /**
+     * @param {Object} requestedMove - Object describing the request to play points
+     * @param {1 | 0} requestedMove.playedBy - Which player is playing
+     * @param { Card } requestedMove.cardPlayed - Card Played for points
+     * @param { MoveType.POINTS } requestedMove.moveType - Specifies that this a Points move
+     */
```

Comment:
Move this above the requestedMove input

Reply:
do we still want this comment? it would just be the req.body now

Reply:
Yes because req.body is the value but it's still helpful to know the shape of data expected in order to understand what you're looking at